go/test/endtoend/utils/mysql.go

```diff
@@ -91,13 +91,13 @@ func NewMySQLWithMysqld(port int, hostname, dbName string, schemaSQL ...string)
 	for _, sql := range schemaSQL {
 		err = prepareMySQLWithSchema(params, sql)
 		if err != nil {
-			return mysql.ConnParams{}, nil, nil, err
+			return mysql.ConnParams{}, nil, nil, "", err
 		}
 	}
 	return params, mysqld, func() {
 		ctx := context.Background()
 		_ = mysqld.Teardown(ctx, mycnf, true)
-	}, nil
+	}, mysqlDir, nil
```

Comment:
Looks like this was added for tests only? It's not used in non-test code paths.

Reply:
this function is defined in the `go/test` package. I don't think it is meant to be used in any non-test code paths.